recipes/promod3/build.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -exo pipefail
+
+# Prevent build failures due to insufficient memory in the CI environment
+# Use parallel build because of serial build on osx-arm64 causing occasional errors
+if [[ "${build_platform}" == "linux-aarch64" || "${build_platform}" == "osx-arm64" ]]; then
+  export CPU_COUNT=$(( CPU_COUNT * 70 / 100))
+  if [ "$CPU_COUNT" -gt 8 ]; then
+    CPU_COUNT=8
+  fi
+  export CPU_COUNT
+  echo "CPU_COUNT for build = $CPU_COUNT"
+fi
```

Comment:
I don't think these lines are needed because there are no heavy GUI-related dependencies as with openstructre.

Reply:
In my experience, heavy compilation of C++ is often problematic, and I have the impression that GUI-related builds often fail.